client/package.json

```diff
@@ -29,11 +29,13 @@
     "yup": "^0.32.8"
   },
   "scripts": {
+    "prepare": "husky install",
+    "prestart": "npm run lint",
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "lint": "eslint \"*/**/*.{js,ts,tsx}\" --fix"
```

Comment:
The double-quote is needed for the command to work in Windows. I imagine Sami's environment needs the double-quote too given that these changes are accidentally being included in his PRs.

Reply:
The original command from the starter works for me in Windows. Are both of you on Powershell? @samimsu

Reply:
Yep. Powershell. But this solution worked without any changes on my side.